src/main/java/org/springframework/data/aerospike/core/AerospikeOperations.java

```diff
@@ -66,18 +66,11 @@
 	MappingContext<?, ?> getMappingContext();
 	
 	/**
-	 * Save operation using the WritePolicy.recordExisits policy of CREATE_ONLY
+	 * Save operation
 	 * @param objectToInsert
 	 */
 	void save(Object objectToInsert);
 	
-	/**
-	 * Save operation using the WritePolicy specified.
-	 * @param objectToInsert
-	 * @param policy
-	 */
-	void save(Object objectToInsert, WritePolicy policy);
-	
```

Comment:
You are right, but save method without WritePolicy will also behave as save or update based on version property of document.
This can be seen in:
```
if (entity.hasVersionProperty()) {
  createOrUpdateUsingCas(objectToInsert, key, bins, entity);
} else {
  client.put(createOrUpdatePolicy, key, bins);
}
```
Otherwise we are going to move this logic to client side which is not quite correct.
Other options like expiration, respondAllOps, durableDelete should be supported as configuration. For example, in spring-data-couchbase expiration for a document can be specified via @Document annotation. The same can be done for other options.